### ISSUE TITLE
Show wrong error position

### ISSUE THREAD

**Issue by Akiyamka** [vuejs/eslint-plugin-vue#468]
**Tell us about your environment**

ESLint: 4.19.1
eslint-plugin-vue: 4.5.0
Node: 8.9.4
package.json
https://gist.github.com/Akiyamka/9584396a7783d1039a378334f019e0d0

If script tag comes first, before template, plugin highlight wrong position of error.
Example:
```vue
<script>
import { mapState } from 'vuex';
import { mapMutations } from 'vuex'

</script>

<template>
  <div>
      {{  }}
  </div>
</template>
```
Error: `Parsing error: Expected to be an expression, but got empty.`
expect highlight here: `{{  }}`
actually happened: `import { _<-- mapMutations } from 'vuex'`

![image](https://user-images.githubusercontent.com/6587585/39410983-6c507278-4c09-11e8-8894-7f9dd6235ef8.png)

**Comment by mysticatea** [comment]
Hi. I apologize for the delay.

It looks like a duplicate of #383.
What is the source of the lint error? (the source is shown at the first of the popup, with enclosed by square brackets (e.g., `[eslint] unexpected token`))

**Comment by mysticatea** [comment]
Closing as a duplicate of #383. Thank you.
